static/js/list_render.js

```diff
@@ -264,7 +264,11 @@ exports.create = function ($container, list, opts) {
     };
 
     widget.set_up_event_handlers = function () {
-        meta.scroll_container = scroll_util.get_list_scrolling_container($container);
+        if (opts.simplebar_container) {
+            meta.scroll_container = ui.get_scroll_element(opts.simplebar_container);
+        } else {
+            meta.scroll_container = scroll_util.get_list_scrolling_container($container);
+        }
```

Comment:
@sahil839 We just want to change this to
```
meta.scroll_container = ui.get_scroll_element(opts.simplebar_container);
```

after ensuring that we are using simplebar in all the places where `list_render` is userd. See https://github.com/zulip/zulip/pull/15650#discussion_r451198363. I think this pr would be a good place do it for you.

Reply:
I know there are couple of places where simplebar is not used, so I should change those to use simplebar right?

Reply:
Yes, thanks!